CustomSuggestionService/ChatModelManagement/ChatModelEditView.swift

```diff
@@ -289,6 +293,40 @@ struct ChatModelEditView: View {
         }
         .padding(.vertical)
     }
+
+    @ViewBuilder
+    var anthropic: some View {
+        baseURLTextField(prompt: Text("https://api.anthropic.com")) {
+            Text("/v1/messages")
+        }
+        apiKeyNamePicker
+
+        TextField("Model Name", text: $store.modelName)
```

Comment:
Please also update `ChatModelEdit.swift` and update the `case .checkSuggestedMaxTokens` for Antrhopic models.

Reply:
Updated!